File: tests/conformance/fixtures/go/21-edge-outputs-v1.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	// v1-style outputs (positional, auto-named)
	p.Task("build").Run("make build").Outputs("dist/app", "dist/lib")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/21-edge-outputs-v1.exs
use Sykli

pipeline do
  task "build" do
    run "make build"
    outputs ["dist/app", "dist/lib"]
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
